pom.xml

```diff
@@ -10,8 +10,8 @@
   <version>2021.06.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>MyCoRe</name>
-  <description xml:space="preserve">
-MyCoRe is a system for the development of institutional content repositories, publication servers, image collections, archives, or similar repositories.
+  <description xml:space="preserve">
```

Comment:
This is probably a linebreak issue - Can you remove this change from PR

Reply:
I have no idea whats happening there. Whenever I save the file (IDE or normal editor), git detects a change.

Reply:
You're right, for whatever reason the is a Windows style linebreak (`0xD 0xA`) after `"preserve">`